tests/integration/targets/azure_rm_vmbackuppolicy/tasks/main.yml

```diff
@@ -0,0 +1,56 @@
+- name: Fix resource prefix
+  set_fact:
+    name: "myPolicy{{ resource_group | hash('md5') | truncate(22, True, '') }}"
+    vault_name: "myVault"
+
```

Comment:
I think the recovery service vault should be created in this sample and deleted at the end.  Please add create recovery vault service at this place.

Reply:
test cases are added